shell/platform/tizen/tizen_window_ecore_wl2.cc

```diff
@@ -216,15 +216,17 @@ void TizenWindowEcoreWl2::RegisterEventHandlers() {
           auto* configure_event =
               reinterpret_cast<Ecore_Wl2_Event_Window_Configure*>(event);
           if (configure_event->win == self->GetWindowId()) {
-            ecore_wl2_egl_window_resize_with_rotation(
-                self->ecore_wl2_egl_window_, configure_event->x,
-                configure_event->y, configure_event->w, configure_event->h,
-                self->GetRotation());
-
-            self->view_delegate_->OnResize(
-                configure_event->x, configure_event->y, configure_event->w,
-                configure_event->h);
-            return ECORE_CALLBACK_DONE;
+            if (configure_event->w != 0 && configure_event->h != 0) {
```

Comment:
Shouldn't it be

```suggestion
            if (configure_event->w != 0 || configure_event->h != 0) {
```

Otherwise resize won't happen if only width or height changes.

Reply:
If both `w` and `h` are 0, the meaning is that there is no change.
If only `w` has been changed, it will be called by the changed `w` and `h`(not 0).

In addition, if one of them is set to 0 and the `resize` is called, no operation will occur. So isn't it okay, "When both are not zero"?

Reply:
I've discussed with @Swanseo0 and @JSUYA offline and concluded that this issue may not be relevant at the moment.